src/format.h

```diff
@@ -22,8 +22,10 @@
 
 namespace amber {
 
+/// The format class describes requested image formats. (eg. R8G8B8A8_UINT).
```

Comment:
I'm not sure what you mean. The format class can describe a VkFormat, but it doesn't have too.

Reply:
I meant that you mentioned only **image formats** in the comment, but it can also describe **vertex attributes** for vertex buffer e.g., position, color, normal.

In addition, I'd use **texel formats of images** rather than **image formats**.